tempo_embeddings/embeddings/vector_database.py

```diff
@@ -360,6 +370,21 @@ def embed_text_batch(self, text_batch: list[str]):
             logging.warning("There is no valid embedding function in this database. Returning None")
         return batch_embeddings
 
+    def get_metadata_stats(self, collection: Collection, include_only: list[str] = None) -> dict[dict[str, int]]:
+        result = collection.get(include=["metadatas"])
+        stats = {}
+        for meta_dict in result["metadatas"]:
+            for field_name, value in meta_dict.items():
+                if include_only is None or field_name in include_only:
+                    if field_name in stats:
+                        if value in stats[field_name]:
+                            stats[field_name][value] += 1
+                        else:
+                            stats[field_name][value] = 1
+                    else:
+                        stats[field_name] = {value: 1}
```

Comment:
Could this be done using a [Counter](https://docs.python.org/3/library/collections.html#collections.Counter)?

I think it should roughly work like this:
```python
        stats = Counter()
        for meta_dict in result["metadatas"]:
            for field_name, value in meta_dict.items():
                if include_only is None or field_name in include_only:
                    stats[field_name] += 1
```
The nested `if`'s probably need to be corrected in my suggestion, but I hope a Counter makes the logic a bit easier to understand.